### PR TITLE
create 4 folders inside containers folder, each folder has an index a…

### PR DESCRIPTION
**Pull request by amonamoqadi**
…nd style files .. create file for variables

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import Container from './containers/Container';
-import Projects from './containers/Projects';
-import About from './containers/About';
-import Contact from './containers/Contact';
+import Container from './containers/NavBar/index';
+import Projects from './containers/Projects/index';
+import About from './containers/About/index';
+import Contact from './containers/Contact/index';
```
Comment (IbrahimLama) [inline src/App.js:5]: it should work without adding index, 
this way 
```import Container from './containers/NavBar';```
check it, and rename Container to NavBar

File: src/containers/About/index.js [hunk 2]
```diff
@@ -1,30 +1,30 @@
 import React from 'react';
-import "../App.scss";
-import AboutImg from '../components/AboutImg';
+import "./style.scss";
+import AboutImg from '../../components/AboutImg';
```
Comment (IbrahimLama) [inline src/containers/About/index.js:3]: import order, 
external libraries, 
components, 
helpers,
styles

File: src/containers/About/index.js [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
 
 function About(){
     const AboutInfo = [
         {
         name:"Jhon Doe",
-        src:require('../img/john.jpg'),
+        src:require('../../img/john.jpg'),
```
Comment (IbrahimLama) [inline src/containers/About/index.js:9]: move the import to the top this way
```import johnImage from '../../img/john.jpg';```
apply this to other images

File: src/App.js (hunk 1 above)
Comment (amonamoqadi) [reply src/App.js:5]: yup, it works

File: src/containers/NavBar/style.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            margin-left: 6px;
+            margin-right: 10px;
+            font-weight: 800;
+        }
+
+        .Panner__Links {
+            display: none;
+        }
+   }
+   .Container__Inner {
```
Comment (IbrahimLama) [inline src/containers/NavBar/style.scss:34]: add line above

File: src/containers/Projects/index.js [hunk 5]
```diff
@@ -1,40 +1,40 @@
...
     },
     {
       name:"Barn House",
-      src:require("../img/barn.jpg")
+      src:require("../../img/barn.jpg")
     }, 
        {
       name:"Summer House",
-      src:require("../img/summer.jpg")
+      src:require("../../img/summer.jpg")
```
Comment (IbrahimLama) [inline src/containers/Projects/index.js:25]: move the import above

**Pull request merged**